src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -2186,8 +2238,7 @@ private bool ReduceBoxSelectionToEditableBox(ref VirtualSnapshotPoint selectionT
                     if (selectionLeftColumn > maxPromptLength || maxPromptLength == minPromptLength)
                     {
                         selectionTopLine = editableLine;
-                        selectionLeftColumn = Math.Max(selectionLeftColumn, maxPromptLength);
-                        result = false;
```

Comment:
My understanding is `ReduceBoxSelectionToEditableBox` returns `false` if the selection can not be reduced to editable box, and it is not true in this case, therefore `result = false;` is removed.